tests/contrib/pytest/test_pytest.py

```diff
@@ -29,6 +29,41 @@ def subprocess_run(self, *args):
         """Execute test script with test tracer."""
         return self.testdir.runpytest_subprocess(*args)
 
+    def test_patch_all(self):
+        """Test with --ddtrace-patch-all."""
+        py_file = self.testdir.makepyfile(
+            """
+            import ddtrace.monkey
+
+            def test_patched_all():
+                assert ddtrace.monkey._PATCHED_MODULES
+        """
+        )
+        file_name = os.path.basename(py_file.strpath)
+        rec = self.inline_run("--ddtrace-patch-all", file_name)
+        rec.assertoutcome(passed=1)
+        spans = self.pop_spans()
+
+        assert len(spans) == 0
+
+    def test_patch_all_init(self):
+        """Test withddtrace-patch-all via ini."""
+        self.testdir.makefile(".ini", pytest="[pytest]\nddtrace-patch-all=1\n")
+        py_file = self.testdir.makepyfile(
+            """
+            import ddtrace.monkey
+
+            def test_patched_all():
+                assert ddtrace.monkey._PATCHED_MODULES
+        """
```

Comment:
```suggestion
            """
```

Reply:
If you want, but it's not the convention in this file.
I don't really care anyhow.

Reply:
it's just a suggestions since the indentation looked inconsistent 🙂